app/Forms/MembersSignupForm.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Forms;
+
+use Kris\LaravelFormBuilder\Form;
+use Carbon\Carbon;
+
+class MembersSignupForm extends Form
+{
+    public function buildForm()
+    {
+        // Basic bio details.
+        $this
+            ->add('first_name', 'text', [
+                'rules' => 'required|min:1',
```

Comment:
I think these rules should be kept in one place, e.g. with the associated model.

You'll want to enforce the same rules as well in similar forms, e.g. if members are allowed to update their profile info, and it'll be nice not have to duplicate the rules all over.